lib/src/wavy.dart

```diff
@@ -233,17 +232,18 @@ class _WTextPainter extends CustomPainter {
 
   void calculateMove() {
     double height = _textLayoutInfo[0].height;
-    double _percent = percent - txtInMoInd;
-    int txtInMoOdd = (percent - .5).floor();
+    double _percent = progress - progress.floor();
```

Comment:
Try switching 235 and 236 around so that you only call `progress.floor()` once; you can reuse `txtInMoInd` to compute `_percent`.

Reply:
Okk will do it

Reply:
Changes are made